Code/Framework/AzCore/AzCore/RTTI/TypeSafeIntegral.h

```diff
@@ -32,6 +32,24 @@ namespace AZStd
     AZ_DEFINE_ENUM_RELATIONAL_OPERATORS(TYPE_NAME)                                                                                                  \
     AZ_DEFINE_ENUM_BITWISE_OPERATORS(TYPE_NAME)
 
+//! This implements AZStd to_string methods for a type safe integral alias.
+//! This must be placed in global scope (not in a namespace) and the TYPE_NAME should be fully qualified.
```

Comment:
There is no need to specify the enum underlying type.

`AZStd::to_underlying` knows how to extract the underlying type from the enum